examples/tutorials/08_research_tutorial/2_Research_advanced.ipynb

```diff
@@ -0,0 +1,1030 @@
+{
```

Comment:
<p>Move path checking and removing to a separate cell and clearly explain why it is needed and why it might be a risky operation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>